Tests/PerformanceTests/wrk.sh

```diff
@@ -0,0 +1 @@
+wrk -t4 -c200 -d30s http://localhost:8080/telegram -s telegram.lua
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Tips depend on target shell and yours is unknown. Add a shebang.](https://app.codacy.com/app/FabrizioBrancati/SwiftyBot/pullRequest?prid=3796373)